index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Succulent Spot</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body class="container">
+    <header id="blue">
```

Comment:
Let's see if we can come up with something more descriptive than `blue`. I think `blue` would work better for a class to clearly show something is going to be blue. But as an id, which is specific and unique, something more descriptive would be good!